tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -257,10 +281,6 @@ func (c *CustomClusterProvisioningTestSuite) ProvisioningRKE2CustomClusterDynami
 					err = wait.WatchWait(result, checkFunc)
 					assert.NoError(c.T(), err)
 					assert.Equal(c.T(), clusterName, clusterResp.ObjectMeta.Name)
-
-					clusterToken, err := clusters.CheckServiceAccountTokenSecret(client, clusterName)
-					require.NoError(c.T(), err)
-					assert.NotEmpty(c.T(), clusterToken)
 				})
 			}
 		}
```

Comment:
since there are new fields to enter in the config file, please also update the readme in the rke2 folder.

Reply:
Done